exchanges/kraken/kraken_wrapper.go

```diff
@@ -330,7 +330,10 @@ func (k *Kraken) Run() {
 // FetchTradablePairs returns a list of the exchanges tradable pairs
 func (k *Kraken) FetchTradablePairs(assetType asset.Item) ([]string, error) {
 	var products []string
-
+	format, err := k.GetPairFormat(assetType, false)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
You could use this formatting in `Futures`, otherwise this was only used in spot and should just go back